.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 1
```

Comment:
is version 1 intentional? Not version 2?

Reply:
Oops - fixed